packages/desktop-client/src/components/reports/reports/CashFlowCard.tsx

```
@@ -138,6 +363,13 @@
       ]}
       onMenuSelect={item => {
         switch (item) {
+          case 'change-view':
+            const newValue = switchFlag ? 'condensed' : 'full';
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)